_data/exhibitions.yml

```diff
@@ -1,17 +1,23 @@
-- title: dotCSS
-  link: https://www.dotcss.io/
-  date: 4. Dec 2019
+- title: ePages Hackathon
+  link:
+  date: 25.-26. Mar 2020
   time:
-  address: Dock Pullmann<br>87 Avenue des Magasins Généraux (Bâtiment 137)<br>93300 Aubervilliers
-
-- title: dotJS
-  link: https://www.dotjs.io/
-  date: 5.-6. Dec 2019
-  time:
-  address: Dock Pullmann<br>87 Avenue des Magasins Généraux (Bâtiment 137)<br>93300 Aubervilliers
+  address: ePages GmbH<br>Pilatuspool 2<br>20355 Hamburg
 
 - title: Stellenwerk Job Fair
   link: https://www.stellenwerk-jobmesse.de/hamburg/en/
   date: 16. Jun 2020
   time:
   address: Campus Universität Hamburg<br>Edmund-Siemers-Alle 1<br>20146 Hamburg
+
+- title: Design Matters
+  link: https://designmatters.io/
+  date: 23.-24. Sept 2020
+  time:
+  address: Lokomotivværkstedet<br>Otto Busses Vej 5A<br>2450 Copenhagen SV
+
+- title: Commerce Summit
+  link:
```

Comment:
Same for this one. Maybe a link to https://developer.epages.com/blog/events/commerce-summit-lets-start-now/ ?